### PR TITLE
UHF-3810 TODOs

### PR DESCRIPTION
**Pull request by teroelonen**
# [UHF-3810](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3810)

## What was done

* Went through all the TODOs in the HDBT-theme and created tickets of the ones that didn't have.
* Removed some old code that was found when checking the TODOs.

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-3810_todos`
* Run `make drush-cr`

## How to test

* [x] Check the TODOs make sense and are written in similar format to help when searching them.
* [x] Check that the service page with service units listed on the bottom works fine if you add some paragraphs to the lower content.
* [x] Check that the unit page with accessibility information accordion on the bottom works fine if you add some paragraphs to the lower content.
* [x] Check that if you add `<nolink>` link to main menu first level it doesn't break to layout like it did before.
* [x] Check that the map paragraph and external video paragraph and image-placeholder still work correctly after the aspect-ratio code change. You can for example add them to a standard page and scale the page to make sure they still scale nicely.
* [x] Check that code follows our standards

## Designers review

* [x] This PR does not need designers review
* [ ] This PR has been visually reviewed by a designer (Name of the designer)

[UHF-3810]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-3810?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Arkkimaagi — CHANGES_REQUESTED** [review]
Looks good, but I found a couple little questions..

File: src/scss/06_components/paragraphs/_content-liftup.scss [hunk 1]
```diff
@@ -74,10 +74,10 @@ $-info-row-icon-size: $spacing-and-half;
 }
 
 .content-liftup__image .image-placeholder {
-  padding-bottom: aspect-ratio-padding($-content-liftup-aspect-ratio-mobile);
+  aspect-ratio: ($-content-liftup-aspect-ratio-mobile);
```
[the remark this answers is not in the document]
Comment (teroelonen) [reply src/scss/06_components/paragraphs/_content-liftup.scss:77]: Good catch! That is incorrect!

File: src/scss/06_components/paragraphs/_content-liftup.scss [hunk 2]
```diff
@@ -74,10 +74,10 @@ $-info-row-icon-size: $spacing-and-half;
...
 
   @include breakpoint($breakpoint-m) {
-    padding-bottom: aspect-ratio-padding($-content-liftup-aspect-ratio-desktop);
+    aspect-ratio: ($-content-liftup-aspect-ratio-desktop);
```
[the remark this answers is not in the document]
Comment (teroelonen) [reply src/scss/06_components/paragraphs/_content-liftup.scss:80]: Fixed this one too.

**Comment by Arkkimaagi** [comment]
Approved, but github mobile has hidden the approve action too well 🙈

**Review by Arkkimaagi — APPROVED** [review]

**Pull request merged**